crates/rune/src/module/module.rs

```diff
@@ -571,17 +577,37 @@ impl Module {
                     handler: data.handler,
                     docs,
                 });
+                &mut self.macros.last_mut().unwrap().docs
             }
-        }
+            MacroMetaKind::Attribute(data) => {
+                let hash = Hash::type_hash(&data.item);
 
-        let m = self.macros.last_mut().unwrap();
-        Ok(ItemMut { docs: &mut m.docs })
+                if !self.names.insert(Name::Macro(hash)) {
```

Comment:
Isn't this supposed to be `Name::AttributeMacro`?

Reply:
ups, musst have missed that one